recipes/python-toolchain/bld.bat

```diff
@@ -0,0 +1,19 @@
+:: This works for `setuptools`, but breaks `distutils`.
+:: Will need to figure out a better long term strategy.
+copy "%RECIPE_DIR%\distutils.cfg" "%STDLIB_DIR%\distutils\distutils.cfg"
+if errorlevel 1 exit 1
+
+:: Configure `pip`.
+IF NOT EXIST "%LIBRARY_PREFIX%\etc" mkdir "%LIBRARY_PREFIX%\etc"
+if errorlevel 1 exit 1
+copy "%RECIPE_DIR%\pip.conf" "%LIBRARY_PREFIX%\etc\pip.conf"
+if errorlevel 1 exit 1
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST "%LIBRARY_PREFIX%\etc\conda\%%F.d" MKDIR "%LIBRARY_PREFIX%\etc\conda\%%F.d"
```

Comment:
Now with the fix in https://github.com/conda-forge/staged-recipes/pull/705#issuecomment-223464170 confirmed to work when the stuff is installed in `"%PREFIX%\etc\conda\%%F.d"`